README.md

```diff
@@ -597,7 +597,7 @@ The Gradle plugin is published to the [Gradle plugin portal](https://plugins.gra
 Given that the `master` branch on the upstream repository is in the state from which you want to create a release, execute the following steps:
 
 1. [Create release via the GitHub UI](https://github.com/ePages-de/restdocs-api-spec/releases/new) <br>
-    Use the intended version number as "Tag version", e.g. "0.15.1".
+    Use the intended version number as "Tag version", e.g. "0.15.4".
```

Comment:
0.15.3, 0.15.3, 0.15.3, 0.15.**4**?

Reply:
Yes, this was my intention. If now we have the "0.15.3" release, if you follow the checklist, you would need to use "0.15.4". But it may be confusing.